client/src/components/NameAndPrice.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {NamePriceContainer, ItemName, ItemPrice} from '../styling.jsx';
+
+const NameAndPrice = (props) => {
+  return (
+    <NamePriceContainer>
+      <ItemName>{props.name}</ItemName>
+      <br></br>
```

Comment:
Setting `display: block;` on the ItemName element may remove the need for this manual linebreak.

Reply:
Oh great call! I'm only scratching the surface with CSS right now, I have a feeling there's a lot of refactoring in my near-future